src/tests/query.rs

```diff
@@ -52,8 +53,14 @@ async fn select_star_from(api: &mut dyn TestApi) -> crate::Result<()> {
     let select = Select::from_table(&table);
     let row = api.conn().select(select).await?.into_single()?;
 
-    assert_eq!(Value::integer(4), row["id"]);
-    assert_eq!(Value::integer(3), row["value"]);
+    // TODO: Make SQLite expect i32 once we differentiate i32 from i64
+    if api.connector_tag().intersects(Tags::SQLITE) {
+        assert_eq!(Value::int64(4), row["id"]);
+        assert_eq!(Value::int64(3), row["value"]);
+    } else {
+        assert_eq!(Value::int32(4), row["id"]);
+        assert_eq!(Value::int32(3), row["value"]);
+    }
```

Comment:
You'll see a lot of those if/then sqlite in the tests. I know this adds a lot of noise but we intend to remove all those SQLite conditions for Prisma 4 when we'll break SQLite.
I could've used `as_integer()` everywhere but hard checking against i32 or i64 was useful to ensure results were as expected.